cms/templates/js/staff-lock-editor.underscore

```diff
@@ -22,4 +22,17 @@
         </p>
     <% } %>
 </div>
+<% if (enableHideFromTOCUI) { %>
+<div class="modal-section-content hide-from-toc">
+<% if (!xblockInfo.isVertical()) { %>
+  <label class="label">
+      <input type="checkbox" id="hide_from_toc" name="hide_from_toc" class="input input-checkbox" />
+      <%- gettext('Hide in Course Outline, accessible via Link') %>
```

Comment:
```suggestion
      <p>
         <%- gettext('Hide in Course Outline, accessible via Link') %>
      </p>
```

Reply:
Thank you! Just so you know – this was already addressed.